examples/js/effects/VREffect.js

```diff
@@ -55,34 +69,76 @@ THREE.VREffect = function ( renderer, onError ) {
 
 	// fullscreen
 
-	var isFullscreen = false;
+	var isPresenting = false;
 
 	var canvas = renderer.domElement;
 	var fullscreenchange = canvas.mozRequestFullScreen ? 'mozfullscreenchange' : 'webkitfullscreenchange';
 
-	document.addEventListener( fullscreenchange, function ( event ) {
+	document.addEventListener( fullscreenchange, function () {
+
+		if ( deprecatedAPI ) {
+
+			isPresenting = document.mozFullScreenElement || document.webkitFullscreenElement;
```

Comment:
I know this is how this code worked before, but shouldn't this actually check for `vrHMD`? because otherwise you could fullscreen anything without a `vrDisplay` or HMD and `isPresenting` would be `true`.